### ISSUE TITLE
Wireless settings different from Tessel 2 hardware

### ISSUE THREAD

**Issue by johnnyman727** [tessel/t2-vm#3]
The VM doesn't have a `/etc/config/wireless` file and lists no wireless devices. Is there anyway we can make the wireless settings from Tessel 2 replicated on the VM? Otherwise the `wifi` CLI commands won't work.

**Comment by kevinmehall** [comment]
VirtualBox doesn't simulate a wireless interface, so this is expected and `tessel wifi` isn't useful for a VM. It's connected to the host via a virtual ethernet interface.  I suppose you could use USB forwarding to pass a USB WiFi adapter into the VM, if you really wanted to.

**Comment by Frijol** [comment]
I would expect people using the VM would generally not try to use `tessel wifi` or would expect it to not work.

**Comment by johnnyman727** [comment]
I think what I'd like to come out of this is just an addition to the README that notes that `wifi` related commands will not work.

**Comment by Frijol** [comment]
in the VM readme?

**Comment by johnnyman727** [comment]
Yeah - the README of this repo. I'm happy to do it once we get the CLI functionality working.
